core_data_modules/cleaners/codes/kenya_codes.py

```diff
@@ -83,6 +83,7 @@ class KenyaCodes(object):
     CHEPALUNGU = "chepalungu"
     CHERANGANY = "cherangany"
     CHESUMEI = "chesumei"
+    CHUKA_IGAMBANG_OMBE = "chuka/igambang'ombe"
```

Comment:
see my mbita comments

Reply:
Const. "Igambang'ombe" is now officially referred as "chuka/igambang'ombe". leaving as is.